scripts/plugins/storage/postgres/init.sql

```diff
@@ -1139,6 +1140,18 @@ INSERT INTO fledge.schedules ( id, schedule_name, process_name, schedule_type,
                 true                                    -- enabled
               );
 
+-- Update alerts
+INSERT INTO fledge.schedules ( id, schedule_name, process_name, schedule_type,
+                                schedule_time, schedule_interval, exclusive, enabled )
+       VALUES ( '852cd8e4-3c29-440b-89ca-2c7691b0450d', -- id
+                'update alerts',                        -- schedule_name
+                'update alerts',                        -- process_name
+                2,                                      -- schedule_type (interval)
+                '00:05:00',                             -- schedule_time
+                '00:00:00',                             -- schedule_interval
```

Comment:
I believe time should be 24:00:00. Please check with Mark.

Reply:
24:00:00 is gives error of invalid time format while running the job.

Reply:
oh yeah, `%H time format covers 00-23`. You may use `23:59:59`